src/main/resources/avro/referencemethods.avdl

```diff
@@ -261,17 +260,89 @@ ListReferenceBasesResponse getReferenceBases(
   /** Additional request parameters to restrict the query. */
   ListReferenceBasesRequest request) throws GAException;
 
-/****************  /sequence/{id}  *******************/
+/******************  /sequences/search  *********************/
+/**
+This request maps to the body of `POST /sequences/search` as JSON.
+*/
+record SearchSequencesRequest {
+  /**
+  If not null, return only `Sequence`s that appear in the indicated
+  `ReferenceSet`, or any included `ReferenceSet`s.
```

Comment:
Shouldn't this composability be made explicit in the definition of `ReferenceSet`? This was news to me.